@uportal/notification-list/src/components/NotificationItem.vue

```diff
@@ -31,9 +31,9 @@
                 </p>
                 <p class="media-link" v-if="notification.url">
                     <a
-                        href="javascript:void(0)"
+                        href=""
```

Comment:
will this force a page refresh? or will `preventDefault` catch this?

Reply:
preventDefault catches it. I need the href or the a tag doesn't get styled correctly.